src/lib/logical_query_plan/abstract_lqp_node.cpp

```diff
@@ -208,6 +209,12 @@ ColumnID AbstractLQPNode::get_column_id(const AbstractExpression& expression) co
   return *column_id;
 }
 
+bool AbstractLQPNode::is_column_nullable(const ColumnID column_id) const {
+  // Default behaviour: Forward from input
+  Assert(left_input() && !right_input(), "Can only from input iff there is a left input and no right input");
```

Comment:
can only... what?

Reply:
covfefe

Reply:
"Can forward nullability from input iff there is a left input and no right input"